src/App.jsx

```diff
@@ -3,7 +3,7 @@ import { Route, Switch, Redirect } from "react-router-dom";
 import { withRouter } from "react-router";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
-import NotFound from "./pages/404/notFound";
+import NotFound from "./pages/404/NotFound";
```

Comment:
✔ correct change.

I think we should merge this one before merging #900 and #899

Reply:
> ✔ correct change.
> 
> I think we should merge this one before merging #900 and #899

I don't Know where is error 
Can you tell me please @ashuvssut?

Reply:
No error in this PR

Reply:
you PR #899 has the error

Reply:
![image](https://user-images.githubusercontent.com/60546840/115281966-76c9b500-a0fe-11eb-8922-4166ceeb1a5a.png)

Reply:
you are adding notFound.jsx

That must be NotFound.jsx